zusammen-commons-db/zusammen-commons-db-impl/src/main/java/com/amdocs/zusammen/commons/db/impl/cassandra/CassandraConnectionFactory.java

```diff
@@ -98,6 +98,11 @@ private static Cluster initCluster() {
             CassandraConfig.getConsistencyLevel().ifPresent(
                     s -> builder.withQueryOptions(new QueryOptions().setConsistencyLevel(ConsistencyLevel.valueOf(s))));
         }
+
+        LOGGER.debug("Creating Cassandra cluster to hosts:{} port:{} with reconnect timeout:{} SSL is: {}",
```

Comment:
LOGGER.debug("Creating Cassandra cluster to hosts:{} port:{} with reconnect timeout:{} SSL is: {}", nodes,
                CassandraConfig.getPort().isPresent() ? CassandraConfig.getPort().get() : "default",
                CassandraConfig.getReconnectionDelay().isPresent() ? CassandraConfig.getReconnectionDelay().get() :
                        "default", CassandraConfig.isSsl());